docs/src/components/file-tree.astro

```diff
@@ -17,7 +17,7 @@ const processedContent = await fileTreeProcessor.process({
 });
 ---
 
-<file-tree set:html={processedContent} />
+<file-tree set:html={processedContent} data-pagefind-ignore="index" />
```

Comment:
Looks like we can leave out the attribute value here according to the docs: https://pagefind.app/docs/indexing/#removing-individual-elements-from-the-index

```suggestion
<file-tree set:html={processedContent} data-pagefind-ignore />
```